app/Services/ImportReport.php

```diff
@@ -17,14 +17,26 @@ public function generateCSV(string $path): void
         // write column headers
         fputcsv($handle, config('imports.report.headers'));
 
+        $res = DB::table('mismatches')
+            ->select('import_id', 'review_status')
+            ->selectRaw('COUNT(*)')
+            ->groupBy('import_id', 'review_status')
+            ->get();
+
+        $review_statuses = config('mismatches.validation.review_status.accepted_values');
+
         foreach ($imports as $each_import) {
-            $mismatches = Mismatch::where('import_id', $each_import->id)->get();
-            $mismatches_count = $mismatches->count();
-            $review_statuses = config('mismatches.validation.review_status.accepted_values');
+            foreach ($review_statuses as $status) {
+                ${"mismatches_" . $status} = 0;
```

Comment:
Not sure about how happy I am about this syntax, over just using an associative array. But not introduced with this PR, so I don't consider this blocking.

Reply:
In particular, I was searching for a while for where `$mismatches_pending` is defined/set 🙄

Reply:
Yeah, I had the same issue when I was working on `missing` support ^^ happy to upload a follow-up change that uses an array instead.

Reply:
yep, let's do that in a follow-up